### PR TITLE
Add pages to the app

### PR DESCRIPTION
**Pull request by Abe1able**
### In this milestone of the project, I have

- created the complete website consisting of 3 pages: Home, Calculator, and Quote.
- added my own styles to improve the look and feel.
- used react router for the navigation.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
+  background-color: #ffdee9;
+  background-image: linear-gradient(90deg, #ffdee9 0%, #b5fffc 100%);
 }
 
 .container {
   width: 420px;
+  padding-left: 50px;
 }
 
 .row-div {
```
Comment (zieeco) [inline src/App.css]: - Hey, please kindly follow the placement of the calculator as shown in the 
[wireframe](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/images/math_magicians_calculator.png) and also add the missing text at the left "Let's do some maths" to meet the wireframe as much as you can. below is your design:


Screenshot from 2022-11-10 22-12-29

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Abe1able ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 2]
```diff
@@ -10,42 +10,47 @@ const Calculator = () => {
...
+          <div className="result">
+            { stateObject.total }
+            { stateObject.operation }
+            { stateObject.next }
+          </div>
+        </div>
+        <div className="row-div">
+          <button type="button" className="cell" onClick={onClickHandler}>AC</button>
+          <button type="button" className="cell" onClick={onClickHandler}>+/-</button>
+          <button type="button" className="cell" onClick={onClickHandler}>%</button>
```
Comment (tufoinnkuo10) [inline src/components/Calculator.js:28]: - [optional] you can make the size of your calculator buttons and calculator size a little bigger so as to make it more user friendly. great work

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @Abe1able 

Your project is complete! Great job for making the requested changes 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- design well followed
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**